### PR TITLE
Features/7898 - Filter Complete

### PR DESCRIPTION
**Pull request by sslaws**
## Status
Ready

## Fixes or Implements [AB#7898](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7898)
- [X] Enhancement
- [ ] Bug
- [ ] Documentation

## Description:
A common filter to dynamically enable/disable a controller or action.  Initially implemented only for Immunization.

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [X] Not Required

### Steps to Reproduce:
If this is a bug, please provide details on how to reproduce the code.

### UI Changes
No

### Browsers Tested
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox

Provide an explanation for any test(s) not completed.
Tests will come when the work item is complete.

## Notes/Additional Comments
Early implementation as this work is complete and can be verified in OpenShift.

### CODE REVIEW

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_330)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_330)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_330)

File: Apps/WebClient/src/appsettings.json [hunk 1]
```diff
@@ -49,5 +49,9 @@
       "Encounter": "false",
       "Note": "false"
     }
+  },
+  "AvailabilityFilter": {
```
Comment (FuriousLlama) [inline Apps/WebClient/src/appsettings.json]: We should merge this and the modules to have a single point of configuration for enabling/disabling modles (medications. imz, etc)

Comment (sslaws) [reply Apps/WebClient/src/appsettings.json]: I'll raise a task for this as the refactoring may not be worth it.  Currently External Configuration is loaded completely from WebClient but this Filter should be at the root level as it applies to all applications.

**Pull request merged**